core/src/test/java/com/google/googlejavaformat/java/MainTest.java

```diff
@@ -251,4 +251,28 @@ public void importRemoveErrorParseError() throws Exception {
     assertThat(main.format("-")).isEqualTo(1);
     assertThat(err.toString()).contains("<stdin>:4:3: error: class, interface, or enum expected");
   }
+
+  @Test
+  public void validate() throws Exception {
+    Main main =
+        new Main(
+            new PrintWriter(System.out, true),
+            new PrintWriter(System.err, true),
+            System.in);
+    int errorCodeOk = main.format("--validate", //
+        tempJava("Simple", "enum Simple {}\n")//
+    );
+    assertThat(errorCodeOk).named("Error Code").isEqualTo(0);
+    int errorCode = main.format("--validate", //
+        tempJava("IsFormatted", "interface IsFormatted {}\n"), //
+        tempJava("NeedsFormatting", "  class  NeedsFormatting  {  }\n") //
+    );
+    assertThat(errorCode).named("Error Code").isEqualTo(1);
+  }
+
+  private String tempJava(String name, String code) throws Exception  {
+    Path path = testFolder.newFile( name + ".java").toPath();
```

Comment:
There's an unneeded space between the bracket and `name`.

May I suggest running `google-java-format` on the files touched by this PR to make sure everything's nice and compliant? :)